source/collection-management/tests/pre_and_post_images-collection.yml

```diff
@@ -0,0 +1,87 @@
+description: "pre_and_post_images-collection"
+
+schemaVersion: "1.9"
+
+runOnRequirements:
+  - minServerVersion: "6.0"
+
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents: [ commandStartedEvent ]
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name papi-tests
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name test
+
+initialData:
```

Comment:
If each test starts with dropping the collection, is there any purpose to using `initialData` here?

Reply:
Nope!